runtime/doc/nvim_clipboard.txt

```diff
@@ -29,4 +35,77 @@ following option:
 See 'clipboard' for details and more options.
 
 ==============================================================================
+2. X11 selection mechanism				    *nvim-clipboard-x11*
```

Comment:
> but the tag should remain *x11-selection*

sure

> I don't know why we're prefixing a bunch of tags with nvim-, I think we need to reconsider that practice...

I think it makes sense in certain cases, such as namespacing headers (e.g., `remote-plugin-(intro|hosts|example|manifest)`, but yeah, I agree it's not needed in this case.